# This is a list of mission status upon mission commander. 
bool algorithm
float32 moving_distance